gapis/resolve/command_tree.go

```diff
@@ -80,10 +83,10 @@ func CommandTreeNode(ctx context.Context, c *path.CommandTreeNode) (*service.Com
 	cmdTree := boxed.(*commandTree)
 
 	switch item := cmdTree.index(c.Indices).(type) {
-	case api.CmdID:
+	case api.SubCmdIdx:
```

Comment:
We get the fully qualified SubCmdIdx (which maybe we can find a better name for it), but most of the time we get 
[CmdID] but if we have a subcommand, we get [CmdId, x, x, x].